android/src/main/java/com/twiliorn/library/CustomTwilioVideoView.java

```diff
@@ -334,6 +334,17 @@ private boolean createLocalVideo(boolean enableVideo, String cameraType) {
         return true;
     }
 
+    private void captureFrame(Promise promise) {
+        cameraCapturer.onFrameCaptured(VideoFrame frame);
```

Comment:
This code doesn't look like it would compile

Reply:
That is quite likely, I was early in piecing together code from various sources when I switched to working on the web implementation.